docs/neuroimaging.md

```diff
@@ -107,16 +175,39 @@ Each person's brain is different. When performing neuroimaging studies, it is im
 ### Tools 
 
 #### Nibabel
-A very powerful python-based tool to access neuro-imaging data (`.nii`, `.dcm` etc.) stored on our file system is [nibabel](https://nipy.org/nibabel/). You can easilly install the latest release of `nibabel` using `pip`:
+
+A very powerful python-based tool to access neuro-imaging data (`.nii`, `.dcm` etc.) stored on our file system is {{ Nibabel }}.
+
+You can easilly install the latest release of {{ Nibabel }} using `pip`:
 
 ```
 pip install nibabel 
 ```
 
+Example of reading a nifti image using {{ Nibabel }}:
+
 ```{code-cell}
+import os
 import nibabel as nib
 
-data = nib.load('example_file.nii')
+data_dir = os.getenv("NILEARN_DATA")
+data = nib.load(data_dir + "/haxby2001/subj2/anat.nii.gz")
```

Comment:
I think this one will be a problem. A regular contributor running `make` will get a `NoneType + str` error because `NILEARN_DATA` will not be set under normal circumstances.

Now, since this section is on loading an arbitrary nifti file using `nibabel`, we should probably not rely on `nilearn` specific stuff to avoid confusion to the reader. Perhaps using just an image of a Shepp-Logan phantom in 2D stored as NIfTI would be enough.

Reply:
Nevermind, I found a way using `nilearn` internals.